api/types.go

```diff
@@ -176,7 +177,7 @@ type Service struct {
 	ForceSsl         bool           `yaml:"force_ssl" json:"force_ssl"`
 	PortMappings     []*PortMapping `yaml:"port_mappings,omitempty" json:"port_mappings,omitempty"`
 	Hosts            []*Host        `yaml:"hosts" json:"hosts"`
-	Listeners        []*Listener    `yaml:"listeners" json:"listeners"`
+	Listeners        []*Listener    `yaml:"listeners,omitempty" json:"listeners,omitempty"`
```

Comment:
why did you add omitempty?

Reply:
To make it optional

Reply:
ok sounds fine. was worried that this might have been added accidentally.